src/components/detector.ts

```diff
@@ -61,12 +60,9 @@ export default class Detector extends EventEmitter implements IDetector {
    * @memberof Detector
    */
   async init(): Promise<any> {
-    const configData = getFramingConfig();
     if (this._options.shouldAutoFrame !== undefined && this._options.shouldAutoFrame !== null) {
-      configData.AUTO_PTZ = this._options.shouldAutoFrame;
+      return this.uploadFramingConfig({ AUTO_PTZ: false });
```

Comment:
Nice @kaiaw

Reply:
@kaiaw this change is included on the latest release right?

Reply:
Yes